src/display/lv_display.c

```diff
@@ -405,7 +405,15 @@ void lv_display_set_buffers(lv_display_t * disp, void * buf1, void * buf2, uint3
 
     lv_color_format_t cf = lv_display_get_color_format(disp);
     uint32_t stride = lv_draw_buf_width_to_stride(w, cf);
-    if(stride * h > buf_size && render_mode != LV_DISPLAY_RENDER_MODE_PARTIAL) {
+    if(LV_DISPLAY_RENDER_MODE_PARTIAL == render_mode) {
```

Comment:
render_mode == LV_DISPLAY_RENDER_MODE_PARTIAL
Follow the overall style of lvgl.

Reply:
done